twitter-as.js

```diff
@@ -0,0 +1,236 @@
+var http = require("http");
+
+var MTwitter = require("./mtwitter.js").MatrixTwitter;
```

Comment:
Generally, the var name should be the same as the class name unless there is a really good reason not to. This is not a good reason.